app/javascript/components/booking/Reserve.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchDevelopers } from "../redux/developer";
+import { ToastContainer, toast } from "react-toastify";
+import "./reserve.css";
+
+import "react-toastify/dist/ReactToastify.css";
+
+function Reserve() {
```

Comment:
- [ ] When I try to reserve a developer, this happens:

![image](https://user-images.githubusercontent.com/57421823/201234749-c3089258-fd9a-4f90-ab72-7ae1adc011ce.png)

It seems that normal users are not authorized to make reservations.  Please, give permission to normal users to reserve developers.  That way, your app will run as intended.  💪